requirements.txt

```diff
@@ -24,3 +24,7 @@ ipython
 factory_boy==3.2.0
 faker==8.11.0
 
+# For linting
+mypy==0.910
```

Comment:
I think I remember seeing `black` used in payment-services, long-term would we want to standardize our linting tools across Django projects?

Reply:
I would be open to that, for sure.